edge-apps/weather/static/js/main.js

```diff
@@ -66,7 +66,16 @@ function initApp (data) {
 
   const countriesUsingFahrenheit = ['US', 'BS', 'KY', 'LR', 'PW', 'FM', 'MH']
   const timeZonesUsingFahrenheit = countriesUsingFahrenheit.map(
-    country => moment.tz.zonesForCountry(country)).flat()
+    country => {
+      let timezones = moment.tz.zonesForCountry(country)
+
+      if (country === "BS") {
+        timezones = timezones.filter(tz => tz !== 'America/Toronto')
```

Comment:
No, no. This is a slippery slope. Please find a library that can do this.

Reply:
Is there a particular reason we want to tie time units to timezone instead of country?

Reply:
No, according to ChatGPT at least, there is no country that uses both. Thus mapping it to country would be fine. Yet, I want a library for this rather than a hard coded list (which can get long i guess).